src/styles/variables.less

```diff
@@ -11,4 +9,39 @@
 
 @knappBredde: 13em;
 
-@navBorder: @navGra20;
+@navBorder: var(--navds-semantic-color-border); // works?
```

Comment:
Does it? :)

Reply:
hehe ops, trodde jag hade fjernet kommentaren. ser bra ut hvertfall